share/ruby-install/util.sh

```diff
@@ -12,7 +12,7 @@ function fetch()
 
 	while IFS="" read -r line; do
 		if [[ "$line" == "$key:"* ]]; then
-			echo "${line##$key:*([[:space:]])}"
+			echo "${line##"$key":*([[:space:]])}"
```

Comment:
Are quotes really needed within bash variable substitutions?

Reply:
I don't think so, but shellcheck disagrees :D

Reply:
Wait, which version of ShellCheck are you using? I'm on 0.7.2, which might be older than your version, which might explain why I'm not seeing those warnings?

Reply:
Mine is 0.8.0 :D